spotbugs/src/main/java/edu/umd/cs/findbugs/FindBugs.java

```diff
@@ -38,7 +38,8 @@
 import edu.umd.cs.findbugs.filter.FilterException;
 import edu.umd.cs.findbugs.internalAnnotations.StaticConstant;
 
-import static java.util.logging.Level.*;
```

Comment:
@juditKnoll spotless is not capable of fixing this 

- https://github.com/diffplug/spotless/pull/2526

Reply:
Just google 'spotless remove star imports' and read the AI response which walks through how to do that.